### PR TITLE
Rollup warnings

### LINKED ISSUE #220: Look into rollup warnings
Rollup current generates 'CIRCULAR_DEPENDENCY' and 'MISSING_GLOBAL_NAME' warnings for the test target.   See if we can get rid of these or at least mute them if they're not an issue.

### PR DESCRIPTION
**Pull request by hildjj**
Fixes #220.

### CODE REVIEW

**Comment by hildjj** [comment]
I'll rebase this once #248 lands.

File: rollup.config.js [hunk 1]
```diff
@@ -49,13 +63,11 @@ const browser_test_config = {
...
       chai: "chai",
-      fs: "ignore_fs",
-      path: "ignore_path",
-      url: "ignore_url",
     },
   },
   external: ["chai"],
   plugins : [
+    ignore(["fs", "os", "path", "tty", "url", "util"]),
     json(),
```
Comment (Mingun) [inline rollup.config.js:71]: Just using 
```js
    globals: {
      // ...
      tty: "ignore_tty",
      url: "ignore_url",
      util: "ignore_util",
    },
```
does not work?

Have you found the reason of those warnings? It seems that these modules used only by CLI, but CLI shouldn't be bundled in browser bundle, isn't it? (I've tried to search in the peggy source tree usages of `"fs"`, `"os"`, `"path"`, `"tty"`, `"url"`, `"util"` and all these strings are used only in tests or CLI or not used at all in the peggy code. The bundled code itself should have no any runtime dependencies, so it is unclear why those warnings is generated in the first place).

Comment (hildjj) [reply rollup.config.js:71]: The `ignore_` approach almost works, but it leaves this warning still:

```js
{
  code: 'MISSING_NODE_BUILTINS',
  message: 'Creating a browser bundle that depends on Node.js built-in modules ("util", "os", "tty", "url" and "path"). You might need to include https://github.com/snowpackjs/rollup-plugin-polyfill-node',
  modules: [ 'util', 'os', 'tty', 'url', 'path' ],
  toString: [Function (anonymous)]
}
```

The warnings that Rollup produces are inadequate to easily track where those modules are being used, but they are probably down in `sinon` or something.  Sinon also has a circular dependency in it, so I've created #253 to see if we can remove it one day.

Comment (hildjj) [reply rollup.config.js:71]: `os` and `tty` at least come from `supports-color`, which `sinon` includes.

Comment (Mingun) [reply rollup.config.js:71]: Ah... I see that these modules are used in `browser_test_config`. Well, maybe they are really not used, at least tests in `development/test.html` pass (the referred file `docs/js/test-bundle.min.js` is a minified version of `build/rollup/test.umd.js`, which is generated using `browser_test_config`).

**Review by Mingun — APPROVED** [review]

**Pull request merged**